x/sequencers/keeper/dymint.go

```diff
@@ -13,40 +13,59 @@ import (
 )
 
 // set dymint sequencers from InitChain
-func (k Keeper) SetDymintSequencers(ctx sdk.Context, validators []abci.ValidatorUpdate) {
-	for _, val := range validators {
-		tmkey, err := tmcrypto.PubKeyFromProto(val.PubKey)
+func (k Keeper) SetDymintSequencers(ctx sdk.Context, sequencers []abci.ValidatorUpdate) {
+	if len(sequencers) == 0 {
+		panic(types.ErrNoSequencerOnInitChain)
+	}
+
+	if len(sequencers) > 2 {
+		panic(types.ErrMultipleDymintSequencers)
+	}
```

Comment:
```suggestion
	if len(sequencers) != 2 {
		panic(types.ErrNoSequencerOnInitChain)
	}

```

Reply:
I want to separate between error cases. anyway refactored it a bit